docs/writing-policies/rust/03-define-policy-settings.md

```diff
@@ -1,14 +1,35 @@
 ---
-sidebar_label: "Defining Policy Settings"
-title: ""
+sidebar_label: "Defining policy settings"
+title: "Defining policy settings"
+description: Defining policy settings for a Kubewarden policy developed using Rust
+keywords: [kubewarden, kubernetes, writing policies, policy settings, rust]
+doc-type: [tutorial]
+doc-topic: [writing-policies, rust, policy-settings]
 ---
 
-# Defining policy settings
+<!--TODO:
 
-As a first step we will define the structure that holds the policy settings.
+These steps are not working for me. I get a few errors on
 
-Open the `src/settings.rs` file and change the definition of the `Settings`
-struct to look like that:
+cargo test
+cargo test
+   Compiling demo v0.1.0 (/home/jhk/projects/suse/tmp/demo)
+error[E0063]: missing field `invalid_names` in initializer of `settings::Settings`
+  \-\-> src/lib.rs:80:23
+   |
+80 |             settings: Settings {},
+   |                       ^^^^^^^^ missing `invalid_names`
+
+Three of those errors. Any suggestions?
+
+-->
```

Comment:
As far as I can see from the error message, your tests need to be updated after you added the field `invalid_names` in the settings struct. The tests seems to be in the previous version of the code when you do not have the `invalid_names` field yet. You need something like this:

```rust
        let tc = Testcase {
            name: String::from("Valid name"),
            fixture_file: String::from(request_file),
            expected_validation_result: true,
            settings: Settings {invalid_names: HashSet::from(["invalid name"])},
        };
```

Reply:
Thanks @jvanz. All good.